intera_examples/scripts/send_random_trajectory.py

```diff
@@ -159,17 +162,15 @@ def main():
             traj.set_log_file_name(args.log_file_output)
 
         if not args.do_not_send:
-            result = traj.send_trajectory()
-
-            if result is None:
-                rospy.logerr('Trajectory FAILED to send')
-            elif result.result:
-                rospy.loginfo('Motion controller successfully finished the trajectory!')
+            result = traj.send_trajectory(timeout=args.timeout)
+            if result is not None and result.result:
```

Comment:
Why did you change the error output here which was not included in #118? It can be helpful to track when it was a network issue versus the trajectory did not complete.

Reply:
I didn't intend to change any functionality, so please correct me if I did. I just cleaned up the messy logging statements (which admittedly wasn't necessary).

Reply:
Ah, I see it now. Yep, these lines were confusing when performing the cherry-pick as I had to merge by hand.

Reply:
There must have been changes other than here, which is causing my confusion. If you compare the `development` version and the `intera-testing-5.1` version side by side, you'll see these lines don't mesh. What is the desired functionality / output here?

Reply:
Keep the change to add the timeout argument parameter to send_trajectory.

The if-else-statement and logging should stay with what was in the development branch.